sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InMemoryFileIndex.scala

```diff
@@ -297,7 +297,7 @@ object InMemoryFileIndex extends Logging {
     val missingFiles = mutable.ArrayBuffer.empty[String]
     val filteredLeafStatuses = allLeafStatuses.filterNot(
       status => shouldFilterOut(status.getPath.getName))
-    val resolvedLeafStatuses = filteredLeafStatuses.flatMap {
+    val resolvedLeafStatuses = filteredLeafStatuses.par.flatMap {
       case f: LocatedFileStatus =>
```

Comment:
There is a comment about `listLeafFilesInParallel` at L311 is not correct as `listLeafFilesInParallel` is removed. Can you also update this comment too?

https://github.com/apache/spark/blob/2bb5924e04eba5accfe58a4fbae094d46cc36488/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/InMemoryFileIndex.scala#L311

Reply:
Thanks. The comment was updated